docs/podman-run.1.md

```diff
@@ -77,6 +77,14 @@ Add Linux capabilities
 
 Drop Linux capabilities
 
+**--cgroup**=*mode*
```

Comment:
`docker` might be going to implement this in a different name such as `--cgroupns-mode` https://github.com/moby/moby/pull/38377

Reply:
I like much more `--cgroup` or `--cgroupns` but it is probably worth to rename it to avoid confusion with Docker.  Thanks to have pointed it out.

Reply:
opened https://github.com/docker/cli/issues/1988 for discussing the flag name